docs/adr/adr_003_bundling_collection_data.md

```diff
@@ -0,0 +1,230 @@
+# Bundling Collection Data for Download
+
+
+* Status: proposed
+* Deciders: mleplatre, gbeckley, acottner
+* Date: June 27 2024
+
+Tracking issue(s):
+-  [The initial remote settings download is inefficient](https://bugzilla.mozilla.org/show_bug.cgi?id=1889617)
+
+
+## Context and Problem Statement
+When a new Firefox client comes online for the first time, it makes over 1,700 HTTP requests to download the initial remote settings data. These are mostly attachments from large collections (ex: `security-state/intermediate, blocklists/certificates`). Opening this many HTTP connections to download small files is not as efficient compared to downloading a few larger files.
+1. It’s a poor user experience if the UI locks up while trying to maintain this many connections
+2. It causes excess power usage and [battery drain](https://bugzilla.mozilla.org/show_bug.cgi?id=1889617) on laptop and mobile devices by keeping modems/wifi devices running for longer than they need to 
+
+If we can bundle these downloads together by collection, we could reduce the number of requests for certain collections from almost thousands to a few. This should reduce the overall time required to download the data (less time establishing connections), which will reduce power usage and improve the user experience.
+
+Note: This improvement is primarily targeting new clients, or clients that are very out of date. Clients that are established and updating daily will not see improvement from this. However, we will be working on new binary replication logic that will help established clients in the future.
```

Comment:
I don't think we should put this here: `However, we will be working on new binary replication logic that will help established clients in the future.`
Because it may never be true

Reply:
That's fair, priorities could shift.